### PR TITLE
chore(FDS-498) [Monorepo] - update typescript

### PR DESCRIPTION
**Pull request by lordpixel**
# Typescript

Version: 4.6.2
Date: 2022-03-04

> [Release notes](https://github.com/microsoft/TypeScript/releases/tag/v4.6.2).

### Ecosystem updates

- none

### Snapshots
Snapshots updated due to aun update in dependencies.

### Dependencies
Dependencies updated due to bump in library versions.

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

File: ADR/rollup.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Rollup
+
+version: 2.67.3
+Date: 2022-02-02
+
+### Features
+
+- Improve side effect detection when using Array.prototype.groupBy/groupByToMap (#4360)
```
Comment (odiseo42) [inline ADR/rollup.md:8]: I think we are using the ADR here to keep a copy of the changelog of a library itself — I would prefer if we keep our ADRs only for those changes, discussions or problems that affect our internal code. 
For instance if there's a library update that introduces API changes that make our code incompatible, that's the kind of information we would want to keep in the ADR.

File: docs/src/lib/MDX_COMPONENTS.js [hunk 2]
```diff
@@ -10,17 +10,17 @@ import {
   AreaPlaceholder,
   Boundaries,
   Button,
-  ChatMessages,
```
Comment (odiseo42) [inline docs/src/lib/MDX_COMPONENTS.js:13]: I'm confused, why is this renaming of components happening in this PR?

**Review by odiseo42 — COMMENTED** [review]
Two pending comments form me in this PR

File: docs/src/lib/MDX_COMPONENTS.js (hunk 2 above)
Comment (lordpixel) [reply docs/src/lib/MDX_COMPONENTS.js:13]: This is not correct. I am investigating, I did not touch this file.

Comment (lordpixel) [reply docs/src/lib/MDX_COMPONENTS.js]: The PR was pointing to `main`, which is incorrect. Branch revisited and correctly set. 👍

**Pull request merged**